scripts/status_consistency.py

```diff
@@ -0,0 +1,36 @@
+from app import geo
+from app.firebase import db
+
+from app.constants import locationsTable, venuesTable, GPS_LOCATIONS
+
+# The casing of tripadvisor is not consistent across our Firebase scripts
+TA_PROVIDER = "tripadvisor"
+TA_DETAILS = "tripAdvisor"
+
+dryRun = True
+
+def updateStatus(center, radius_km, provider, detailsProvider, version):
```

Comment:
nit: What does `updateStatus` mean? e.g. How exactly does the status get updated? Can you add a fn comment or clarify by changing the fn name?